doc/build-osx.md

```diff
@@ -24,7 +24,7 @@ be re-done or updated every time Xcode is updated.
 There's also an assumption that you already have `git` installed. If
 not, it's the path of least resistance to install [Github for Mac](https://mac.github.com/)
 (OS X 10.7+) or
-[Git for OS X](https://code.google.com/p/git-osx-installer/). It is also
+[Git for OS X](http://sourceforge.net/projects/git-osx-installer/files/). It is also
```

Comment:
Yes. Agreed with removing the git section (or mention something like "git is required and will be installed with a recent version of Xcode").